common/src/main/java/org/apache/rocketmq/common/MixAll.java

```diff
@@ -518,4 +520,25 @@ public static boolean isSysConsumerGroupForNoColdReadLimit(String consumerGroup)
         }
         return false;
     }
+
+    public static Set<String> invalidProperties(Properties properties, Object... objects) {
+        if (objects == null || objects.length == 0) {
+            return null;
+        }
+        Set<String> propertyNames = properties.stringPropertyNames();
+
+        Set<String> fieldNames = new HashSet<>();
+        for (Object obj : objects) {
+            Method[] methods = obj.getClass().getMethods();
+            for (Method method : methods) {
+                String mn = method.getName();
+                if (mn.startsWith("get")) {
+                    // get getter's field name and to camel name.
+                    fieldNames.add(mn.substring(3, 4).toLowerCase() +  mn.substring(4));
+                }
+            }
+        }
+
+        return Sets.difference(propertyNames, fieldNames);
```

Comment:
I think it may miss logging when propertyNames not contains all fieldNames

Reply:
I'm a little confused. The difference method  only return a set of property in property name set (which is user configure) but not in field names set (which is RocketMQ Configuration Item). So I think it will correctly record the invalid property name set in user's configuration file.
I have some test case in ``MixAllTest.java``, It would be  better explain that when logging will be missed if you could give me some clearly test case.~

Reply:
Gotcha！there are tests failures, plz fix it :)
<img width="1270" alt="image" src="https://github.com/apache/rocketmq/assets/52153761/5489eb96-f0a9-4bd1-943d-452642772770">